pkg/controller/job/job_controller_test.go

```diff
@@ -980,28 +1016,38 @@ func TestControllerSyncJob(t *testing.T) {
 			expectedCreations:       0,
 			expectedDeletions:       2,
 			expectedActive:          0,
-			expectedCondition:       &jobConditionSuspended,
-			expectedConditionStatus: v1.ConditionTrue,
-			expectedConditionReason: "JobSuspended",
-			expectedPodPatches:      2,
-			expectedReady:           ptr.To[int32](0),
-			expectedTerminating:     ptr.To[int32](2),
+			expectedConditions: []batch.JobCondition{
+				{
+					Type:    batch.JobSuspended,
+					Status:  v1.ConditionTrue,
+					Reason:  "JobSuspended",
```

Comment:
Should we have a const for "JobSuspended" also as we have for others? It is out of scope for this PR tho

Reply:
I think we should yes, but it would entail API review to put it together with other ones, so I would defer for another dedicated PR.